Assets/MirageXR/Player/Scripts/Augmentations/Model/Model.cs

```diff
@@ -143,8 +138,11 @@ private void OnFinishLoadingAsync(GameObject model, AnimationClip[] clip)
                 animation.clip.legacy = true;
                 animation.Play();
             }
-            gameObject.AddComponent<BoundsControl>();
-            EditModeChanges(_activityManager.EditModeActive);
+
+            if (GetComponentInParent<PoiEditor>())
+            {
+                GetComponentInParent<PoiEditor>().EnableBoundsControl(true);
+            }
```

Comment:
the `GetComponentInParent` operation is not the easiest and you should try to avoid calling it

```suggestion
            var poiEditor = GetComponentInParent<PoiEditor>();
            if (poiEditor )
            {
                poiEditor.EnableBoundsControl(true);
            }
```